src/components/ChatEntry.js

```diff
@@ -1,22 +1,36 @@
 import React from 'react';
+// import TimeStamp from './TimeStamp';
 import './ChatEntry.css';
 import PropTypes from 'prop-types';
 
 const ChatEntry = (props) => {
+  const heart = props.liked ? '🤍'  : '❤️';
+  console.log(heart)
+
+  const handleLikeButton = () => {
+    props.updateLike(props.id);
+  };
+
   return (
     <div className="chat-entry local">
-      <h2 className="entry-name">Replace with name of sender</h2>
+      <h2 className="entry-name">{props.sender}</h2>
       <section className="entry-bubble">
-        <p>Replace with body of ChatEntry</p>
-        <p className="entry-time">Replace with TimeStamp component</p>
-        <button className="like">🤍</button>
+        <p>{props.body}</p>
+        <p className="entry-time">{props.timeStamp}</p>
```

Comment:
There are a couple things to look at around the time:
- This prop is empty so the time never shows, how is this data being passed from ChatLog to ChatEntry? What would we need to update for this data to flow through like we expect?
- The TimeStamp component takes props.timeStamp and converts it to a text representation. There is a test that looks for `expect(screen.getByText(/\d+ years ago/)).toBeInTheDocument()` which finds the text generated by a TimeStamp instance. How could we create an instance of TimeStamp here and pass it a prop containing `props.timeStamp`?